client/src/Utils/Testing/MockThemeProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { MuiThemeProvider } from '@material-ui/core/styles';
+
+import theme from 'styles/theme';
+
+
+const MockFormProvider : React.FC<Props> = (props) => {
+    return (
+        <MuiThemeProvider theme={theme}>0
```

Comment:
why 0 in the end of the row?

Reply:
whoops , fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1305/commits/b8b99daa1bdb74b882f72fadfca0193a35b4d049)